commit abca98058926f8a97af274ab990bf5ae4475c0cf
Author: Kefu Chai <kchai@redhat.com>
Date:   Thu Jul 5 14:51:21 2018 +0800

    common: extract SubProcess impl into .cc
    
    cleanup the header dependencies and speed-up the compiling
    
    Signed-off-by: Kefu Chai <kchai@redhat.com>

diff --git a/src/mgr/StandbyPyModules.cc b/src/mgr/StandbyPyModules.cc
index 4909c5d9d1d..60fe4a01747 100644
--- a/src/mgr/StandbyPyModules.cc
+++ b/src/mgr/StandbyPyModules.cc
@@ -14,6 +14,7 @@
 #include "StandbyPyModules.h"
 
 #include "common/debug.h"
+#include "common/errno.h"
 
 #include "mgr/MgrContext.h"
 #include "mgr/Gil.h"
